crawl-ref/source/dat/descript/de/branches.txt

```diff
@@ -200,16 +200,16 @@ Ein ausgeklügelter Komplex, der tollkühne Abenteurer auf der Suche nach
 Reichtum und Ruhm anlockt. Wer ihn betritt, muss sich einen Weg durch mehrere
 Kampfarenen bahnen, von denen jede einen Schatz enthält, der von gefährlichen
 und manchmal exotischen Kreaturen bewacht wird. Am Ende wartet ein
-furchterregender Minotaur, der einen großen Haufen Beute von den Leichen
-törichter Abenteurer bewacht.
+furchterregender Minotaur, der einen großen Haufen Beute bewacht, die
+von den Leichen törichter Abenteurer geplündert hat.
```

Comment:
Should it be "die er..."?

Reply:
Yes, it should! I'll fix it in the next commit.